include/vcml/debugging/gdbserver.h

```diff
@@ -35,10 +35,19 @@ enum gdb_signal {
     GDBSIG_KILL = 9,
 };
 
+struct gdb_target {
+    u64 tid;
+    u64 pid;
+    string xml;
+    vector<const cpureg*> cpuregs;
+    target* tgt;
```

Comment:
Could this be a reference? i.e. `target& tgt`?

Reply:
done